tests/Unit/IO/Test_H5.py

```diff
@@ -0,0 +1,86 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+from spectre import DataStructures
+from spectre.IO.H5 import IOH5File
```

Comment:
ping on this

Reply:
I don't think this is possible given the current way we are adding python modules, I believe that every import statement has to come in the form `from spectre.PATH import Foo`, although I don't completely understand this.  Would you be comfortable with something like `from spectre import Spectre_H5`

Reply:
Oh this is just standard Python syntax to give an imported module a name, like `import numpy as np`.

Reply:
Oh, yeah, I just meant that the import statement `import spectre.IO.H5 as spectre_h5` is not feasible in the current form because then every time we would  still have to call `spectre_h5.IOH5File.File() ` every time we wanted to call the `File()` function, which doesn't seem to be what we were looking for.   I guess my point was maybe the module should just be named something different in order to match the syntax of the `DataStructures`  import above.